pkg/controller/bookkeepercluster/bookie.go

```diff
@@ -113,6 +113,16 @@ func makeBookiePodSpec(bk *v1alpha1.BookkeeperCluster) *corev1.PodSpec {
 			},
 		})
 	}
+	ledgerDirs := "/bk/ledgers"
+	journalDirs := "/bk/journal"
+	if bk.Spec.Options["ledgerDirectories"] != "" {
+
+		ledgerDirs = bk.Spec.Options["ledgerDirectories"]
+
+	}
+	if bk.Spec.Options["journalDirectories"] != "" {
```

Comment:
Would be good to add UTs to cover this code.

Reply:
Done